### ISSUE TITLE
Installation and usage guide

### ISSUE THREAD

**Issue by tsarevpp** [bkaradzic/GENie#370]
Hello, I think it would be useful to write some how-to to install the downloads and how to use GENie.
I mean, I just follow the building tutorial of bgfx and i got stuck here because for OS X it just gives you one file and there are no instructions on what to do with it...
Still getting my `bash command not found`. i tried running the downloaded file but it says `bash: Permission denied` even when used with `sudo -s`

**Comment by bkaradzic** [comment]
If you clone bgfx, bimg, and bx, you can just type `make` and it will show you options for building it.

What's the exact issue you're getting?

**Comment by tsarevpp** [comment]
Hi @bkaradzic,
this is not an issue I guess, its just a **suggestion** to write a guide on **how to install and use GENie**.
I am following the steps in the [bgfx's build selection](https://bkaradzic.github.io/bgfx/build.html), but i got stuck at the following line:
```
genie --with-combined-examples --xcode=osx xcode9
```
Just because GENie is not installed on my machine (Console outputs a command not found message). so I came here in the repo and there is no installation/usage guide. I've downloaded the compiled binary for OS X and its just a file named genie without any extension. How can I run the command above? I figured that this file is the executable for GENie, but trying to run it with root user somehow still gives me `bash: Permission denied`.
